community/sharing.md

```diff
@@ -0,0 +1,90 @@
+# Sharing your Mods
+Author(s): code3z (@code3 on the forum)
+
+Once you've created a mod that you're proud of, the next step is to share it.
+Sharing will be a big help to other people. You can also get feedback and help
+from the community. But it may also mean you have to respond to bugs in your
+code and help other people use your code.
+
+## Managing and Sharing Mods using Git
+
+You can use hosted Git repos such as GitHub and Gitlab to upload, manage, share,
```

Comment:
*Git repo(sitory) hosting services*. And GitLab also has two uppercase letters in its name.